tests/harness.c

```diff
@@ -385,10 +385,19 @@ BootManager *prepare_playground(PlaygroundConfig *config)
                 goto fail;
         }
 
-        if (m->sysconfig->root_uuid) {
-                free(m->sysconfig->root_uuid);
+        if (m->sysconfig->root_device) {
+                cbm_probe_free(m->sysconfig->root_device);
         }
-        m->sysconfig->root_uuid = strdup("DUMMY_UUID");
+        m->sysconfig->root_device = calloc(1, sizeof(struct CbmDeviceProbe));
+        if (!m->sysconfig->root_device) {
+                DECLARE_OOM();
+                abort();
+        }
+        /* Create fake root device */
+        CbmDeviceProbe *root = m->sysconfig->root_device;
+        *root = (CbmDeviceProbe){
+                .dev = 0, .part_uuid = strdup("DUMMY_UUID"), .uuid = strdup("UUID"),
```

Comment:
So most testing isn't going anywhere but it would add a little coverage of the else case if you run this twice, once without part_uuid and once with.